package/network/config/firewall/files/firewall.config

```diff
@@ -61,6 +62,7 @@ config rule
 	option proto		udp
 	option src_ip		fc00::/6
 	option dest_ip		fc00::/6
+	option src_port		547
```

Comment:
This is known to break some DHCPv6 servers, see https://lists.openwrt.org/pipermail/openwrt-tickets/2015-August/079306.html for an explanation.

Reply:
Just dug up the RFC, and I can see where the confusion is. I'll update my patch.

> 5.2. UDP Ports
>
>   Clients listen for DHCP messages on UDP port 546.  Servers and relay
>   agents listen for DHCP messages on UDP port 547.